changelog/README.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 24th January 2024
+* Updated documentation of [Get all reservations (ver 2023-06-06)](../operations/reservations.md#get-all-reservations-ver-2023-06-06) to reflect actual state:

```

Comment:
I think this would be clear enough that the behavior hasn't changed:
```suggestion
* Fixed description of [Get all reservations (ver 2023-06-06)](../operations/reservations.md#get-all-reservations-ver-2023-06-06):
```